nym-api/nym-api-requests/src/models.rs

```diff
@@ -115,19 +116,41 @@ pub struct MixnodeStatusResponse {
     pub status: MixnodeStatus,
 }
 
-#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema)]
+#[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema, ToSchema)]
 pub struct NodePerformance {
+    #[schema(value_type = String)]
     pub most_recent: Performance,
+    #[schema(value_type = String)]
     pub last_hour: Performance,
+    #[schema(value_type = String)]
     pub last_24h: Performance,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
+#[derive(ToSchema)]
+#[schema(title = "MixNodeDetails")]
+pub struct MixNodeDetailsSchema {
```

Comment:
nitpicking: this doesn't seem to be used anywhere, perhaps we should remove it then?
unless you wanted to use it for some schemar placeholder

Reply:
corrected, tnx